tests/test_morph.py

```diff
@@ -53,24 +155,26 @@ def test_get_2(self):
 def test_MorphHelper_cache_1(nm_load):
     nodes = Mock()
     nodes.get.side_effect = ['morph-A', 'morph-A', 'morph-B', 'morph-A']
-    test_obj = test_module.MorphHelper('morph-dir', nodes)
-    nm_load.side_effect = Mock
-    morph1 = test_obj.get(1)
-    # should get cached object for 'morph-A'
-    assert test_obj.get(2) is morph1
-    # should get new object ('morph-B')
-    assert test_obj.get(3) is not morph1
-    # 'morph-A' was evicted from cache
-    assert test_obj.get(4) is not morph1
+    with patch.object(test_module.MorphHelper, '_is_biophysical', return_value=True):
+        test_obj = test_module.MorphHelper('morph-dir', nodes)
+        nm_load.side_effect = Mock
+        morph1 = test_obj.get(1)
```

Comment:
I thought indexing in SONATA starts from 0? I understand it does not matter here as we use mocks but can we use indexing from 0, please?

Reply:
good catch !